src/__tests__/HeaderContainer.test.js

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
```

Comment:
Create a test for the whole header instead of testing its subcomponents.

Reply:
What about testing only `src/components/Header/HeaderContainer.component.jsx`? It already contains its subcomponents.